site/content/blog/getting-started-with-ytt.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Getting Started With ytt, Part 1"
+slug: getting-started-with-ytt
+date: 2022-10-12
+author: Varsha Munishwar
+excerpt: "Are you new to ytt and wondering where to start?
+Here is an easy step-by-step tutorial that can introduce you to ytt and help you get started quickly.
```

Comment:
nit: insert a comma between [coordinate adjectives](https://prowritingaid.com/grammar/1008103/Should-I-use-a-comma-between-two-adjectives#:~:text=You%20should%20use%20a%20comma,you%20can%20swap%20their%20order.)
consider:
> ...easy, step-by-step...

nit: using stronger, clearer language to get the point across
consider:
> ...that introduces ytt and gets you started quickly.